src/xdocs/config_imports.xml

```diff
@@ -1578,6 +1591,24 @@ import org.abego.*;
 
 import java.util.Set; //  Wrong order for 'java.util.Set' import.
 
+public class SomeClass { ... }
+        </source>
+
+        <p>
+          To configure the Check with groups of static imports:
+        </p>
+
+        <source>
+&lt;module name=&quot;ImportOrder&quot;&gt;
+  &lt;property name=&quot;staticGroups&quot; value=&quot;org,java&quot;/&gt;
+  &lt;property name=&quot;sortStaticImportsAlphabetically&quot; value=&quot;true&quot;/&gt;
+&lt;/module&gt;
+        </source>
+
+        <source>
+import static org.abego.treelayout.Configuration.AlignmentInLevel; // Group 1
+import static java.lang.Math.abs; // Group 2
```

Comment:
please add here "com" import, to make it clear that it will be placed in group3(default/all-other)

Reply:
done